tidy3d/components/data/data_array.py

```diff
@@ -70,21 +73,19 @@ def __init__(self, data, *args, **kwargs):
         """Initialize ``DataArray``."""
 
         # initialize with untraced data
-        data_untraced = getval(data)
-        super().__init__(data_untraced, *args, **kwargs)
-
-        # if the passed data has tracers, store them in attrs dict
         if isbox(data):
+            super().__init__(data._value, *args, **kwargs)
```

Comment:
Thought it was redundant because `getval()` only checks whether `data` is `Box` and then returns its `._value`, so we are checking `isbox()` twice basically.

Reply:
yea I think what worries me is that the underscore `_value` makes this seem like something we need to be careful about using, in case it changes for whatever reason. the `getval()` is public-facing API so it might be safer, even if it calls `isbox()` twice?

Reply:
Yup you're right. I changed the init according to your other suggestions so it's back to the old behavior now anyways :)

Reply:
Yup you're right. I changed the init according to your other suggestions so it's back to the old behavior now anyways :)